common/djangoapps/course_modes/admin.py

```diff
@@ -216,5 +227,24 @@ class CourseModeExpirationConfigAdmin(admin.ModelAdmin):
     class Meta(object):
         model = CourseModeExpirationConfig
 
+
+class EnrollmentModeAdmin(admin.ModelAdmin):
```

Comment:
Added the admin table just for easy viewing of the options (useful to make sure the migration ran correctly). Here it is (user staff has access): https://cahrens.sandbox.edx.org/admin/course_modes/enrollmentmode/